### PR TITLE
Initial work on profile page

### PR DESCRIPTION
**Pull request by vbertoletti**
Add profile page markup and some styles so it'll be easier once fetching data from backend.

### CODE REVIEW

File: client/js/screens/About/styles.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
+<<<<<<< HEAD
```
Comment (10thfloor) [inline client/js/screens/About/styles.js]: Oops. You missed these merge markers.

File: client/js/screens/LogIn/LogIn.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+              </View>
+            )}
+          </Form>
+        </View>
+      </ImageBackground>
+    </Fragment>
+  );
+};
+
+export default Login;
```
Comment (10thfloor) [inline client/js/screens/LogIn/LogIn.js]: Looks great! Nice job on the Final Form implementation.

File: client/js/screens/Profile/Profile.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import React from "react";
+<<<<<<< HEAD
```
Comment (10thfloor) [inline client/js/screens/Profile/Profile.js]: More merge markers. Remove these carefully.

File: client/js/screens/Profile/ProfileContainer.js [hunk 4]
```diff
@@ -4,12 +4,18 @@ import propTypes from 'prop-types'
 
 class ProfileContainer extends Component {
   static navigationOptions = {
+<<<<<<< HEAD
```
Comment (10thfloor) [inline client/js/screens/Profile/ProfileContainer.js]: Merge markers in your code.

File: mvmnt-lab-api/package.json [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "mvmnt-lab-api",
   "version": "1.0.0",
+<<<<<<< HEAD
```
Comment (10thfloor) [inline mvmnt-lab-api/package.json]: Merge markers.

**Review by 10thfloor — COMMENTED** [review]
Hey Vanessa. Not sure what happened, but there are a lot of unresolved merges in your code. Be careful not to commit code that is unmerged!

**Review by 10thfloor — CHANGES_REQUESTED** [review]
Remove all unmerged code.

**Comment by vbertoletti** [comment]
Hi Mack, conflicts and page improvements should be ok now for your next review. 
Thanks!

**Review by 10thfloor — APPROVED** [review]

**Pull request merged**